src/patterns-hmrc-govuk-team/government-gateway/index.njk

```diff
@@ -11,16 +11,14 @@ layout: twoColumnPage.njk
   Use this pattern when you're linking to a service that requires the user to sign in using Government Gateway.
 </p>
 
-<h2 class="govuk-heading-l">Where to place</h2>
+<h2 class="govuk-heading-l">How to design this pattern</h2>
 
-<p class="govuk-body">You should place this pattern before the link to the service, unless there's a <a href="/patterns-hmrc-govuk-team/" class="govuk-link">design pattern</a> that says differently.</p>
+<p class="govuk-body">You should place this pattern before the link to the service, unless there's a <a href="/patterns-hmrc-govuk-team/" class="govuk-link">content pattern</a> that says differently.</p>
```

Comment:
Small thing but there's a straight quote in the word 'there's'. Could we change to a curly one?

Reply:
Fixed.